docs/docs/general/02-Getting Started/getting-started.md

```diff
@@ -42,12 +50,8 @@ npm i -D askui typescript ts-node @types/jest ts-jest jest
 
 </details>
 
-If you don't like Jest, feel free to use another test framework, e.g., [Jasmine](https://jasmine.github.io/) or [Mocha](https://mochajs.org/).
-
-Now, we are ready to write our first test.
-
 ## Access Token
 
-Currently, you can use our lib without any configuration. But in the future, as we need to prevent misuse of our API, we may need you to create some credentials through our [askui user portal](https://app.askui.com/) (still, totally free) and [configure our library to use these credentials](../../api/Configuration/askui-ui-control-client#credentials) for authenticating and authorizing with our API.
+Currently, you can use our lib without any configuration. In the future, as we need to prevent misuse of our API, we may need you to create some credentials through our [askui user portal](https://app.v2.askui.com/) (while still free) and [configure our library to use these credentials](https://docs.askui.com/docs/api/Configuration/askui-ui-control-client#credentials) for authenticating and authorizing with our API.
 
-Please see [our documentation on how to signup and create the credentials](../08-askui%20User%20Portal/signup.md).
+Please see [our documentation on how to signup and create the credentials](https://docs.askui.com/docs/general/askui%20User%20Portal/signup).
```

Comment:
I thought absolute links inside the docs where a bad practice as you know depend on the domain... (saw that in a lot of places) Why do we use them now?

Reply:
I can not find out how this one changed. Must be some kind of Find&Replace error from my side.

The absolute links at some points in the docs are necessary when you want to link to an anchor (heading usually). That is not supported with markdown (at least not in Docusaurus 😢 )